pkg/test/vpc_client/bastion.go

```diff
@@ -111,38 +115,57 @@ func (vpc *VPC) PrepareBastionProxy(zone string, cidrBlock string, keypairName s
 
 	insts, err := vpc.AWSClient.ListInstances([]string{}, filters...)
 	if err != nil {
-		return nil, err
+		return nil, "", "", err
 	}
 	if len(insts) == 0 {
 		log.LogInfo("Didn't found an existing bastion, going to launch one")
-		if cidrBlock == "" {
-			cidrBlock = CON.RouteDestinationCidrBlock
-		}
-		_, _, err = net.ParseCIDR(cidrBlock)
-		if err != nil {
-			log.LogError("CIDR IP address format is invalid")
-			return nil, err
-		}
 
-		userData := fmt.Sprintf(`#!/bin/bash
+		userData := `#!/bin/bash
```

Comment:
Let's make the userData generation to another function? It's too big.

Reply:
Fixed.Use generateShellCommand() func to generate userData.